README.md

```diff
@@ -317,10 +317,11 @@ Restart postgresql (`sudo systemctl restart postgresql.service`) and in the logs
 pg_partman master background worker master process initialized with role zabbix
 ```
 
-Connect as `zabbix` database user to the `zabbix` database and create the `pg_partman` extension as part of the `public` schema on the `zabbix` database.
+Connect to the `zabbix` database and create the `pg_partman` extension as part of the `public` schema on the `zabbix` database. The following commands should be run by a **superuser** of the database cluster (i.e. 'postgres' is used in this example, suitable for Debian/Ubuntu).
```

Comment:
Yes you are right. Because the `zabbix` user, if done according to the steps I suggested will not be a superuser of the DB. Thank you for addressing this detail.